src/test/kotlin/sirius/db/testutil/MongoMocks.kt

```diff
@@ -8,7 +8,7 @@
 
 package sirius.db.testutil
 
-import io.mockk.spyk
+import org.mockito.Mockito.spy
```

Comment:
**Question:** But we still want to be able to use MockK?? Why is this change necessary?